contracts/RewardsPool.sol

```diff
@@ -168,12 +171,12 @@ contract RewardsPool is IRewardsPool, Initializable, OwnableUpgradeable, Constan
             indexerDeployment.index[deploymentId] = indexerDeployment.unclaim;
         }
 
-        pool.labor[indexer] += amount;
+        pool.labor[indexer] += laborAdd;
```

Comment:
it's better if not add this indexer when labor is 0

Reply:
Is it any difference between use `if` to skip add, and add 0?